### DISCUSSION TITLE
Test with Oanda CFD SPXUSD Symbol

### DISCUSSION THREAD

**Discussion by antonma** [kernc/backtesting.py#861] [Q&A]
Hi All,

I'm am trying to implement a backtest for Oanda CFD SPX Symbol. 
![image](https://user-images.githubusercontent.com/13572239/210282609-b1caddec-fb7a-4b42-9910-0b01f1b7b13e.png)
![image (1)](https://user-images.githubusercontent.com/13572239/210282620-07592a34-786d-41ac-aca4-3a4643523b56.png)
![image (2)](https://user-images.githubusercontent.com/13572239/210282623-afe1e7dd-c75f-4bd4-83b3-6f34eced52ee.png)

Oanda works with units (and not with lots like other Forex Broker) and it is possible to use one decimal point .
So the questions is how can I mimic this symbol in backtesting py the best. 
I use the following already:
bt = Backtest(data, Cluster, cash=cash_g, exclusive_orders=True, margin=0.05 )

margin = 0.05 => 1:20 leverage but I don't know how to handle the decimal problem...

Thank you and best regards,
Anton

**Comment by kernc** [comment]
No support for fractional units. See https://github.com/kernc/backtesting.py/issues/134 for a workaround.
